phpBB/styles/prosilver/template/ucp_notifications.html

```diff
@@ -31,7 +31,7 @@ <h2>{TITLE}</h2>
 								<!-- IF notification_types.EXPLAIN --><br />&nbsp; &nbsp;{notification_types.EXPLAIN}<!-- ENDIF -->
 							</td>
 							<!-- BEGIN notification_methods -->
-								<td class="mark"><input type="checkbox" name="{notification_types.TYPE}_{notification_types.notification_methods.METHOD}"<!-- IF notification_types.notification_methods.AVAILABLE and notification_types.notification_methods.SUBSCRIBED --> checked="checked"<!-- ENDIF --><!-- IF not notification_types.notification_methods.AVAILABLE --> disabled="disabled"<!-- ENDIF --> /></td>
```

Comment:
Doesn't this end up with a possibility where the notification method is checked, and disabled, so it can not be unchecked?

Reply:
Yes! and I want to have a control over it...

Reply:
I just want to control the `checked` & `disabled` , separately.

Reply:
And anyone who does not use the extension is left with a checkbox that might be checked but can not be unchecked?

Reply:
Without the extension, and before this PR,
If a notification is `available` then the checkbox will not be `disabled` & user can check or uncheck as per their preferences.
If a notification is not `available` then the checkbox will be `disabled` & forced uncheck.

Without the extension, and after this PR,
If a notification is `available` then the checkbox will not be `disabled` & user can check or uncheck as per their preferences.
If a notification is not `available` then the checkbox will be `disabled` & user can not check or uncheck.

So what are you trying to tell me about "might be checked but can not be unchecked",
If I need to some corrections then let me know.

Best regards 👍

Reply:
A new commit is made to keep the same behaviour,
But that logic is moved from template to PHP.

Best regards 👍